### PR TITLE
Add registers view and EX of mov instruction

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: cpu.cpp [hunk 1]
```diff
@@ -354,15 +388,80 @@ void Cpu::operandFetch()
...
+            emit PmMDR(true, MDR, true);
+            break;
+        }
+
+        qDebug() << "EX MOV I1";
+        break;
+    }
+    case 2: {
+        memory[ADR] = MDR & 0xFF;
+        memory[ADR + 1] = (u8)MDR >> 8;
```
Comment (rmoldo) [inline cpu.cpp]: You should cast the whole shift as you will lose the high order byte of MDR by casting it to u8

File: cpuwindow.cpp [hunk 2]
```diff
@@ -220,4 +220,130 @@ void CPUwindow::connectBackend()
...
+                break;
+            default:
+                break;
+            }
+        }
+        else {
+            this->ui->GRlineR->setStyleSheet("color: rgb(0, 0, 0);");
+            this->ui->RGview->setStyleSheet("color: rgb(0, 0, 0);");
+
+            this->ui->r0->setStyleSheet("color: rgb(0, 0, 0);");
```
Comment (rmoldo) [inline cpuwindow.cpp:315]: Set the values of registers to 0

**Review by rmoldo — CHANGES_REQUESTED** [review]

File: cpu.cpp (hunk 1 above)
Comment (ghost) [reply cpu.cpp]: I'd like to remove cast, as it doesn't generate any warning

File: cpuwindow.cpp (hunk 2 above)
Comment (ghost) [reply cpuwindow.cpp:315]: This is quite hard. We want to maintain registers to their values when deactivating lines (signals). Reset should be done only when reassembling. Emitting signal on constructor has no effect, so here a much more complex approach is needed. I'd postpone it for later, but as you consider.

Comment (ghost) [reply cpuwindow.cpp:315]: Done

**Review by rmoldo — APPROVED** [review]

**Pull request merged**